libs/design/button/src/button/stroked/stroked.component.ts

```diff
@@ -0,0 +1,55 @@
+import { NgIf } from '@angular/common';
+import {
+  Component,
+  ViewEncapsulation,
+  ChangeDetectionStrategy,
+  HostBinding,
+} from '@angular/core';
+
+import { DaffPrefixSuffixModule } from '@daffodil/design';
+import { DAFF_LOADING_ICON_COMPONENTS } from '@daffodil/design/loading-icon';
+
+import { DaffButtonBaseDirective } from '../button-base.directive';
+
+/**
+ * DaffStrokedButtonComponent is a rectangular outlined button with no background color.
+ *
+ * ## Usage
+ * ```html
+ * <button daff-stroked-button>
+ *  <div daffPrefix></div>
+ *  Stroked Button
+ *  <div daffSuffix></div>
+ * </button>
+ *
+ * <a href="/" daff-stroked-button>
+ *  <div daffPrefix></div>
+ *  Linked stroked button
+ *  <div daffSuffix></div>
+ * </a>
+ * ```
+ */
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
+  selector: '' +
+    'button[daff-stroked-button]' + ',' +
+    'a[daff-stroked-button]',
+  templateUrl: '../button-base.component.html',
+  styleUrl: './stroked.component.scss',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: true,
+  imports: [
+    NgIf,
+    DaffPrefixSuffixModule,
+    DAFF_LOADING_ICON_COMPONENTS,
+  ],
+})
+export class DaffStrokedButtonComponent
+  extends DaffButtonBaseDirective {
```

Comment:
Since we define the `DaffButtonBaseDirective` as a directive, not a component, can we just add that directive as a host directive?

Reply:
I don't think it'll work with `DaffPrefixDirective` and `DaffSuffixDirective` not being able to be used as a hostDirective.

Reply:
Yep. Thanks for the reminder. From empirical testing and https://github.com/angular/angular/issues/51599 it looks like you can't access content/view children in host directives.